README.md

```diff
@@ -1,32 +1,22 @@
 # gRPC for .NET
 
-## Preview out!
+## Available now on .NET Core 3.0!
 
-A preview of gRPC for ASP.NET Core is on [NuGet](https://www.nuget.org/packages/Grpc.AspNetCore.Server). A template using the preview [shipped with .NET Core 3.0 Preview 3](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-core-3-0-preview-3/).
+gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can run anywhere. gRPC enables client and server applications to communicate transparently, and simplifies the building of connected systems.
 
-See https://github.com/grpc/grpc for the official version of gRPC C# (ready for production workloads).
+gRPC functionality for .NET Core 3.0 includes:
 
-## The plan
+* [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCore) &ndash; An ASP.NET Core framework for hosting gRPC services. gRPC on ASP.NET Core integrates with standard ASP.NET Core features like logging, dependency injection (DI), authentication and authorization.
+* [Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client) &ndash; A gRPC client for .NET Core that builds upon the familiar `HttpClient`.
+* [Grpc.Net.ClientFactory](https://www.nuget.org/packages/Grpc.Net.ClientFactory) &ndash; gRPC client integration with `HttpClientFactory`.
 
-We plan to implement a fully-managed version of gRPC for .NET that will be built on top of ASP.NET Core HTTP/2 server.
-Here are some key features:
-- API compatible with the existing gRPC C# implementation (your existing service implementations should work with minimal adjustments)
-- Fully interoperable with other gRPC implementations (in other languages and other platforms)
-- Good integration with the rest of the ASP.NET Core ecosystem
-- High-performance (we plan to utilize some of the cutting edge performance features from ASP.NET Core and in the .NET platform itself)
-- We plan to provide a managed .NET Core client as well (possibly with limited feature set at first)
+Please note that gRPC for .NET does not replace [gRPC for C#](https://grpc.io/docs/quickstart/csharp/) (gRPC C# API over native C-core binaries). These implementations coexist; there are currently no plans for one implementation to replace the other one. gRPC for C# is the recommended solution for frameworks that gRPC for .NET does not support such as .NET Framework and Xamarin.
```

Comment:
pehaps it's better to link directly to https://github.com/grpc/grpc/tree/master/src/csharp (which has a README.md and provides access to the code and also links to the examples) instead of https://grpc.io/docs/quickstart/csharp/